apps/aragon-fundraising/app/src/screens/Reserves.js

```diff
@@ -61,79 +60,80 @@ export default () => {
-  const [opened, setOpened] = useState(false)
-
-  const theme = useTheme()
-  const { layoutName } = useLayout()
-
-  const editMonthlyAllocationButton = <Button icon={<img src={EditIcon} />} label="Update fees" onClick={() => setOpened(true)} />
+  const adjustedBuyFeePct = percentageFromBase(buyFeePct, PCT_BASE)
+  const adjustedSellFee = percentageFromBase(sellFeePct, PCT_BASE)
 
   return (
     <>
-      <Split
-        primary={
-          <>
-            <Box heading="Collateralization ratios">
-              <div
-                css={`
-                  display: grid;
-                  grid-column-gap: ${3 * GU}px;
-                  grid-template-columns: repeat(${layoutName === 'small' ? '1' : '2'}, 1fr);
-                  width: 100%;
-                `}
-              >
-                {[
-                  [primaryCollateralSymbol, primaryCollateralRatio],
-                ].map(([symbol, ratio], i) => (
-                  <ReserveSetting
-                    key={i}
-                    label={`${symbol} collateralization ratio`}
-                    helpContent={helpContent(primaryCollateralSymbol)[0]}
-                    value={
-                      <span>
-                        {ratio}
-                        <span
-                          css={`
-                            margin-left: ${0.5 * GU}px;
-                            color: ${theme.surfaceContentSecondary};
-                          `}
-                        >
-                          %
-                        </span>
-                      </span>
-                    }
-                  />
-                ))}
-              </div>
-            </Box>
-          </>
-        }
-        secondary={
+      <div
+        css={`
+          display: grid;
+          grid-template-columns: auto 1fr 1fr;
+          grid-gap: ${2 * GU}px;
+        `}
+      >
+        <Box heading="Collateralization ratios">
+          <div
+            css={`
+              display: grid;
+              grid-column-gap: ${3 * GU}px;
+              grid-template-columns: repeat(${layoutName === 'small' ? '1' : '2'}, 1fr);
+              width: 100%;
+            `}
+          >
+            {[[primaryCollateralSymbol, primaryCollateralRatio]].map(([symbol, ratio], i) => (
+              <ReserveSetting
+                key={i}
+                label={`${symbol} collateralization ratio`}
+                helpContent={helpContent(primaryCollateralSymbol)[0]}
+                value={
+                  <span>
+                    {ratio}
+                    <span
+                      css={`
+                        margin-left: ${0.5 * GU}px;
+                        color: ${theme.surfaceContentSecondary};
+                      `}
+                    >
+                      %
+                    </span>
+                  </span>
+                }
+              />
+            ))}
+          </div>
+        </Box>
+
+        <div>
```

Comment:
`Box` components direct siblings set a `margin top` by default. This enclosing div is to prevent Boxes to be direct siblings and prevent setting margins

Reply:
Maybe leave that as a comment in the code too.